packages/components/psammead-content-shift-blocker/src/index.test.jsx

```diff
@@ -0,0 +1,398 @@
+/* eslint-disable no-underscore-dangle */
+
```

Comment:
Brilliant work with writing tests for all the different cases. 💯

Reply:
Now this set of tests is a good advocate for the testing trophy approach re my previous comment: https://github.com/bbc/psammead/pull/2582#discussion_r350858961 

Reading them carefully, I can see clearly what they are simulating in the browser, I'm disagreeing with myself :anguished: Either way, these tests are cool :sunglasses: